src/router.ts

```diff
@@ -170,11 +174,10 @@ export class Router {
     const balance = await connection.getTokenAccountBalance(mmOptionAccount);
 
     this.dips[this.dip_to_string(expirationSec, strike)] = {
-      splToken: splMintToToken(splMint),
-      premiumAsset: "USD",
+      baseAsset: splMintToToken(baseMint),
+      quoteAsset: splMintToToken(quoteMint),
       expirationMs: expirationSec * 1_000,
       strike: strike,
```

Comment:
Should I use adjustStrike here too? Do we risk collisions of optionMints if we create a DIP of the same strike & expiration but different quote/base (call & put)?

Reply:
To avoid collision, change dip_to_string 
I think that is only used as a key for this.